installer/PowerToysSetup/Product.wxs

```diff
@@ -710,7 +710,7 @@
         <Condition>INSTALLDESKTOPSHORTCUT</Condition>
         <RegistryValue Root="HKCU"
                         Key="Software\[Manufacturer]\[ProductName]"
-                        Name="desktopshorcutinstalled"
+                        Name="desktopshortcutinstalled"
```

Comment:
Can you remove this one from this PR? Thanks.

Reply:
I've moved the commit to here: https://github.com/jsoref/PowerToys/tree/spelling-shortcut -- It sounds like you can't change this w/o creating orphans. (I pretty much expected that.)

Reply:
@enricogior: can you confirm that you wouldn't want to take a PR for this?